<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EasyMode</class>
 <widget class="QMainWindow" name="EasyMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>933</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyMode</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 168, 82)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(94, 189, 120);
font-family: Arial;
padding: 4px;
color: white;</string>
    </property>
    <property name="text">
     <string>Player Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(94, 189, 120);
font-family: Arial;
padding: 4px;
color: white;</string>
    </property>
    <property name="text">
     <string>Start/Restart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DealButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(94, 189, 120);
font-family: Arial;
padding: 4px;
color: white;</string>
    </property>
    <property name="text">
     <string>Deal Cards</string>
    </property>
   </widget>
   <widget class="QLabel" name="hearts2label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cards/2_of_clubs.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="diamonds2label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cards/2_of_diamonds.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="clubs2label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cards/4_of_diamonds.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="spades2label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>70</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cards/6_of_spades.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="diamonds3label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="clubs3label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="spades3label_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>230</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hearts4label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="diamonds4label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>390</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="clubs4label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>390</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="spades4label_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>390</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hearts5label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>560</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="diamonds5label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>550</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="clubs5label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>550</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="spades5label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>550</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dealtcardlabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>740</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cards/4_of_hearts.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="gamestate">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>760</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Game ongoing...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>832</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
